### ISSUE TITLE
[Alpha 14] PMDG 737 isolation bus constant screen reader message

### ISSUE THREAD

**Issue by Orinks** [jfayre/talking-flight-monitor-net#98]
When First Officer Pro turns on the isolation bus, TFM constantly sends that it is on to the screen reader, making TFM impossible to use after that point. There should be an option to disable the status of various switches/items as there is the instrumentation.

**Comment by ajborka** [comment]
Fixed in Alpha 15 series.
